example-app/src/main/java/com/hcaptcha/example/MainActivity.java

```diff
@@ -65,6 +68,7 @@ private HCaptchaConfig getConfig() {
                 .size(size)
                 .loading(loading.isChecked())
                 .hideDialog(hideDialog.isChecked())
+                .expirationTimeout(10)
```

Comment:
Worth documenting this in [the table under `hideDialog`](https://github.com/hCaptcha/hcaptcha-android-sdk#config-params).